### PR TITLE
Ep 3175 concurrent update

### PR DESCRIPTION
**Pull request by ancashoria**

### CODE REVIEW

File: packages/earth-map/src/components/collection/collection-conflict/CollectionConflict.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          onClick={() => {
+            setIsOverwriting(true);
+            onOverwrite();
+          }}
+          disabled={isOverwriting}
+        >
+          {isOverwriting ? t('Saving') : t('Save anyway')}
+        </button>
+      </Modal>
+      <div id="inner-modal"></div>
```
Comment (corinacioloca) [inline packages/earth-map/src/components/collection/collection-conflict/CollectionConflict.tsx]: what is this?

Comment (ancashoria) [reply packages/earth-map/src/components/collection/collection-conflict/CollectionConflict.tsx]: it's a way to render the modal in the sidebar

File: packages/earth-map/src/components/collection/collection-rename/CollectionRename.tsx [hunk 2]
```diff
@@ -17,33 +17,41 @@
...
 
 export function CollectionRename(props: IProps) {
-  const { collection, onCancel } = props;
+  const { collection, onCancel, reloadCollection, toggleRenaming } = props;
+  const { id, slug, name, organization, version } = collection;
   const { t } = useTranslation();
-  const { id, name, organization } = collection;
   const [saveError, setSaveError] = useState('');
-  const { register, errors, handleSubmit, formState } = useForm({
+  const [isSaveConflict, setIsSaveConflict] = useState(false);
```
Comment (corinacioloca) [inline packages/earth-map/src/components/collection/collection-rename/CollectionRename.tsx:44]: pls run linter

File: packages/earth-map/src/components/sidebar/sidebar-layout-search/styles.scss [hunk 3]
```diff
@@ -29,6 +29,20 @@
...
+    .collection-edit-places {
+      .ReactModal__Overlay {
+        top: 110px;
+      }
+    }
+
+    // Collection save conflict modal
+    .collection-rename {
+      .ReactModal__Overlay {
+        top: 68px;
```
Comment (corinacioloca) [inline packages/earth-map/src/components/sidebar/sidebar-layout-search/styles.scss]: why 68 and not 70? :))

**Review by corinacioloca — APPROVED** [review]

**Pull request merged**